perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -1004,36 +1004,44 @@ sub garden_modify : Chained('garden_setup') : Args(0) {
         $c->detach( '/auth/redirect' );
     }
 
-    my $pick = $c->get_param('task') || '';
-    if ($pick eq 'cancel') {
-        $c->res->redirect('/waste/' . $c->stash->{property}{id} . '/garden_cancel');
-        $c->detach;
-    }
+    if ($c->stash->{garden_sacks}) {
+        my $payment_method = 'credit_card';
+        $c->forward('check_if_staff_can_pay', [ $payment_method ]); # Should always be okay here
```

Comment:
could this not be after the if given it's the same in both conditions? I guess it doesn't matter really.

Reply:
Yeah, thought it was worth keeping separate given it's not actually doing anything here.